cmd/amp/main.go

```diff
@@ -67,7 +67,7 @@ func main() {
 		}
 		AMP = client.NewAMP(Config)
 		if AMP.Verbose() == false {
-			RootCmd.SilenceErrors = true
+			;RootCmd.SilenceErrors = true
```

Comment:
Yes, missed this here, actually fixed it in a subsequent PR, though. Anyway, will fix here, thanks.